Libraries/LibGUI/Model.h

```diff
@@ -44,6 +44,13 @@ enum class SortOrder {
     Descending
 };
 
+class ModelClient {
+public:
+    virtual ~ModelClient() { }
+
+    virtual void on_model_update(unsigned) = 0;
```

Comment:
Same here, possibly even more important since there's no reference implementation to directly check.
```suggestion
    virtual void on_model_update(unsigned flags) = 0;
```